models/issue_mail.go

```diff
@@ -39,11 +39,11 @@ func mailIssueCommentToParticipants(e Engine, issue *Issue, doer *User, content
 
 	// In case the issue poster is not watching the repository and is active,
 	// even if we have duplicated in watchers, can be safely filtered out.
-	poster, err := getUserByID(e, issue.PosterID)
+	issue.loadPoster(e)
```

Comment:
It doesn't seem like you are checking the returned error

Reply:
Oops. Sorry.